mfa/callbacks/internationalCallbacks/messageFailed.md

```diff
@@ -0,0 +1,38 @@
+|:----------------------|:---------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| type                  | `string` | The Callback type. For message failed, this is set to `message-failed`                                                                                                                                                                                                                                                                                                                                                    |
+| time                  | `string`  | The time of the event described in the receipt                                                                                                                                                                                                                                                                                                                      |
+| description           | `string` | A detailed description of the event described by the receipt                                                                                                                                                                                                                                                                                                        |
+| platform           | `string` | A field that indicates whether the message DLR was from the 2FA Domestic or International platform. For International, this is set to `GlobalSMS`.                                                                                                                                                                                                                                                                                             |
+| message           | `Object` | An object that represents information about the message sent.                                                                                                                                                                                                                                                                                             |
+| message.id            | `string` | The unique ID of this message                                                                                                                                                                                                                                                                                           |
```

Comment:
Huge nit - some of these have '.' at the end and some don't, could be nice to standardize.

Reply:
updated!